inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/redis/RedisSink.java

```diff
@@ -0,0 +1,129 @@
+import lombok.experimental.SuperBuilder;
+import org.apache.inlong.manager.common.consts.SinkType;
+import org.apache.inlong.manager.common.util.CommonBeanUtils;
+import org.apache.inlong.manager.common.util.JsonTypeDefine;
+import org.apache.inlong.manager.pojo.sink.SinkRequest;
+import org.apache.inlong.manager.pojo.sink.StreamSink;
+
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Redis sink info
+ */
+@Data
+@SuperBuilder
+@AllArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@ApiModel(value = "Redis sink info")
+@JsonTypeDefine(value = SinkType.REDIS)
+public class RedisSink extends StreamSink {
+
+    @ApiModelProperty("Redis cluster mode")
+    private String clusterMode;
+
+    @ApiModelProperty("Redis database id")
+    private Integer database;
+
+    @ApiModelProperty("Redis data type")
+    private String dataType;
+
+    @ApiModelProperty("Redis schema mapping mode")
+    private String schemaMapMode;
+
+    @ApiModelProperty("Password for Redis accessing")
+    private String password;
+
+    @ApiModelProperty("Database name")
+    private String databaseName;
+
+    @ApiModelProperty("Expire time of Redis row")
+    private Integer ttl;
+
+    @ApiModelProperty("The timeout of Redis client")
+    private Integer timeout;
+
+    @ApiModelProperty("The socket timeout of redis client")
+    private Integer soTimeout;
+
+    @ApiModelProperty("The max total of sink client")
+    private Integer maxTotal;
+
+    @ApiModelProperty("The max idle of sink client")
+    private Integer maxIdle;
+
+    @ApiModelProperty("The min idle of sink client")
+    private Integer minIdle;
+
+    @ApiModelProperty("The max retry time")
+    private Integer maxRetries;
+
+    @ApiModelProperty("The host of Redis server")
+    private String host;
+
+    @ApiModelProperty("The port of Redis server")
+    private Integer port;
+
+    @ApiModelProperty("The master name of Redis sentinel cluster")
+    private String sentinelMasterName;
+
+    @ApiModelProperty("The sentinels info of Redis sentinel cluster")
+    private String sentinelsInfo;
+
+    /**
+     * The address of redis server, format eg: 127.0.0.1:8080,127.0.0.2:8081 .
+     * If server is not cluster mode, server address format eg: 127.0.0.1:8080 .
+     */
+    @ApiModelProperty("The cluster nodes of Redis cluster")
+    private String clusterNodes;
+
+    @ApiModelProperty("The DataEncoding of Redis STATIC_PREFIX_MATCH data-type")
+    private String formatDataEncoding;
+
+    @ApiModelProperty("The DataType of Redis STATIC_PREFIX_MATCH data-type")
+    private String formatDataType;
+
+    @ApiModelProperty("Whether ignore parse error of Redis STATIC_PREFIX_MATCH data-type")
+    private Boolean formatIgnoreParseError;
+
+    @ApiModelProperty("The data separator of Redis STATIC_PREFIX_MATCH data-type")
+    private String formatDataSeparator;
+    @ApiModelProperty("Extended properties")
+    private List<HashMap<String, String>> extList;
```

Comment:
Why not use `StreamSink.properties`? Why is `extList` used here, but `properties` used in RedisSinkDTO?